deploy/0061_deploy_catalysts.ts

```diff
@@ -6,13 +6,19 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const {deployments, getNamedAccounts} = hre;
   const {deploy} = deployments;
 
-  const {deployer} = await getNamedAccounts();
+  const {catalystMinter, deployer} = await getNamedAccounts();
   for (const catalyst of catalysts) {
     await deploy(`Catalyst_${catalyst.name}`, {
       contract: 'PrimaryCatalyst',
       from: deployer,
       log: true,
-      args: [catalyst.name, catalyst.symbol, deployer, catalyst.maxGems],
+      args: [
+        catalyst.name,
```

Comment:
name do not follow what is describe for gems
the best would be remove name from the data file and do 
name : `Sandbox's <symbol> Catalysts`
symbol: `<symbol>`